src/main/java/net/minecraftforge/common/extensions/IForgeBlock.java

```diff
@@ -920,4 +920,30 @@ default MaterialColor getMapColor(BlockState state, BlockGetter level, BlockPos
     {
         return defaultColor;
     }
+
+    /**
+     * Returns the factor by which jump height is multiplied for any entities inside this block or standing on this block (unless overridden by the block the entity is in).
+     *
+     * @param state The state of this block
+     * @param getter The level this block is in
+     * @param pos The block's position in the level
+     * @param entity The entity for which this jump factor is applied
+     */
+    default float getJumpFactor(BlockState state, BlockGetter level, BlockPos pos, Entity entity)
+    {
+        return self().getJumpFactor();
+    }
+
+    /**
+     * Returns the factor by which movement speed is multiplied for any entities inside this block or standing on this block (unless overridden by the block the entity is in).
```

Comment:
Make this multiline, javadocs are smart enough to know how to interpret it. My general statement is that it should wrap after half a screen.